src/Session/SessionManager.cc

```diff
@@ -555,6 +543,39 @@ void SessionManager::RunApp() {
         .run();
 }
 
+bool SessionManager::EnableScripting() {
```

Comment:
Not sure about this function name. I think `IsScriptngEnabled` makes more sense and is consistent with the rest of our codebase (`IsAuthenticated`, `IsValidFrontendFolder`, `IsValid`, `IsCompressedFits`)

Reply:
This is actually unused (a relic of a previous design). I will remove it.